docs/manual/docs/cmd/user/user-get.md

```diff
@@ -0,0 +1,49 @@
+# spo user get
```

Comment:
This file is in the wrong folder. It should be in `docs/manual/docs/cmd/spo/user`